src/boehm_gc.c

```diff
@@ -566,43 +576,57 @@ UInt ResizeBag (
 
 
 /*****************************************************************************
-** The following functions are not required by boehm, so empty implementations
-** are provided
+** The following functions are not required respectively supported, so empty
+** implementations are provided
 **
 */
 
-void            InitMsgsFuncBags (
-    TNumMsgsFuncBags    msgs_func )
-{ }
-
 void InitGlobalBag (
     Bag *               addr,
     const Char *        cookie )
 { }
 
-void SortGlobals( UInt byWhat )
+void CallbackForAllBags(
+     void (*func)(Bag) )
+{ }
+
+void            InitCollectFuncBags (
+    TNumCollectFuncBags before_func,
+    TNumCollectFuncBags after_func )
 { }
 
-Bag * GlobalByCookie(
-       const Char * cookie )
+void SwapMasterPoint( Bag bag1, Bag bag2 )
 {
-    return (Bag *) 0;
+    Obj *ptr1 = PTR_BAG(bag1);
+    Obj *ptr2 = PTR_BAG(bag2);
+    PTR_BAG(bag1) = ptr2;
+    PTR_BAG(bag2) = ptr1;
```

Comment:
Note to self: This function now is not empty anymore, and also is required by other code (and thus by necessity supported), so it should be moved up, to not stand under the above comment.